pkg/overlay/overlay.go

```diff
@@ -153,14 +185,8 @@ func CleanupMount(contentDir string) (Err error) {
 func CleanupContent(containerDir string) (Err error) {
 	contentDir := filepath.Join(containerDir, "overlay")
 
-	if unshare.IsRootless() {
-		mergeDir := filepath.Join(contentDir, "merge")
-		if err := unix.Unmount(mergeDir, 0); err != nil {
-			if !os.IsNotExist(err) {
-				return errors.Wrapf(err, "unmount overlay %s", mergeDir)
-			}
-		}
-	}
+	_ = Unmount(contentDir)
```

Comment:
Is it intentional to ignore the error?

Reply:
yes, on errors `Unmount()` itself will print the warnings

Reply:
It doesn't look like an error from `ioutil.ReadDir()` will be logged.

Reply:
do we still have to handle that somehow or just attempt to delete the directory in any case?

Reply:
I'd expect it to be logged if the caller's just going to discard it anyway.

Reply:
I've changed the implementation to only ignore `EINVAL` when the path is not a mount point